### PR TITLE
[TECHNICAL SUPPORT] LPS-184533 When toggle controls off you can rename a widget

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hey,
https://issues.liferay.com/browse/LPS-184533

I made the portlet title non-editable when toggle control is off.
Please review it.

Thanks,

### CODE REVIEW

**Comment by NemethNorbert** [comment]
@liferay-frontend can you review it please?

File: modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_controls.scss [hunk 1]
```diff
@@ -119,4 +119,8 @@ body.portlet {
 	.portlet-topper {
 		display: none !important;
 	}
+
+	.portlet-title-editable {
+		pointer-events: none;
```
Comment (izaera) [inline modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_controls.scss:124]: :thinking: This disables mouse events, but if people are using keyboard, can they still activate the item?

Comment (izaera) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_controls.scss:124]: @NemethNorbert :point_up: ?

Comment (dsanz) [reply modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_controls.scss:124]: Looks like the portlet name is not under the tab order. Haven't thoroughly checked this but if we confirm that's the case, this solution would be enough. @NemethNorbert could you please confirm this?

**Comment by NemethNorbert** [comment]
Hey @dsanz, @izaera 

Can we move forward with the PR?

**Comment by dsanz** [comment]
ci:forward

**Pull request closed without merging**